src/ai_provider/google_cloud_ai_provider.py

```
@@ -0,0 +1,31 @@
+import os
+from google.cloud import aiplatform
+from .ai_provider import AIProvider
+
+class GoogleCloudAIProvider(AIProvider):
+
+    def __init__(self, project_id, location):
+        aiplatform.init(
+            project=os.environ.get("GCP_PROJECT"),
+            location=os.environ.get("GCP_LOCATION"),
+            credentials=os.environ.get("GCP_CREDENTIALS"),
+        );
```

Ruff

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)